src/Rector/AbstractRector.php

```diff
@@ -119,6 +120,11 @@ public function beforeTraverse(array $nodes): ?array
         return parent::beforeTraverse($nodes);
     }
 
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition('', []);
+    }
+
```

Comment:
We'll still need a way to enforce adding this method in our rules. This would let it silently pass